test/index.html

```diff
@@ -15,7 +15,7 @@
         <script src="../reqs/contextmenu/leaflet.contextmenu-src.js"></script>
         <script src="../reqs/loading/Control.Loading.js"></script>
         <script src="../reqs/markercluster/leaflet.markercluster-src.js"></script>
-        <script src="../reqs/label/leaflet.label-src.js"></script>
+        <script src="../reqs/label/leaflet.label.js"></script>
```

Comment:
Done in https://github.com/umap-project/Leaflet.Storage/commit/5a56290ae3c4b49cd05500b52305c0efadb415a1